### PR TITLE
Spryker Crons

### PR DESCRIPTION
**Pull request by kierenevans**
Run the Spryker crons without Jenkins.

### CODE REVIEW

File: spryker/usr/local/share/spryker/spryker_functions.sh [hunk 1]
```diff
@@ -119,6 +119,7 @@ do_spryker_install() {
     do_spryker_import_demodata
     do_spryker_product_label_relations_update
     do_spryker_setup_search
+    do_spryker_app_permissions
     do_spryker_run_collectors
```
Comment (andytson-inviqa) [inline spryker/usr/local/share/spryker/spryker_functions.sh]: wondering if we should remove this line if we are to rely on crons for dev too

Comment (kierenevans) [inline spryker/usr/local/share/spryker/spryker_functions.sh]: Would mean waiting up to a minute to see something on Yves after the cron container is deployed.

Comment (andytson-inviqa) [inline spryker/usr/local/share/spryker/spryker_functions.sh]: ok

File: spryker/Dockerfile.tmpl [hunk 2]
```diff
@@ -18,4 +18,5 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x > /tmp/install-node.sh \
  && rm -rf /var/lib/apt/lists/*
 
 COPY ./{{WEB_SERVER}}/ /
+COPY ./etc/ /etc/
```
Comment (andytson-inviqa) [inline spryker/Dockerfile.tmpl]: ah, build.sh not run

**Pull request merged**